src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    private String description;
+    private Boolean isDone;
+    protected String type;
+
+    public Task(String description) {
```

Comment:
IMO this constructor should take in `type` as a parameter as well because `type` is declared in this class.